### PR TITLE
Don't let exit()'s within modules stop the daemons.

### PR DESCRIPTION
**Pull request by jacksontj**
This change catches those and just logs the error instead of exiting.

### CODE REVIEW

**Comment by cachedout** [comment]
Good call, but could we change the error messages? Showing this as a syntax error could be confusing.

**Comment by jacksontj** [comment]
Sure, one sec.

**Comment by cachedout** [comment]
Sweet thanks!

File: salt/modules/test.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 __proxyenabled__ = ['*']
 
-
+exit(1)
```
Comment (thatch45) [inline salt/modules/test.py]: This will just fill up the logs, can we make a module in the integration test suite that does this instead?

Comment (jacksontj) [inline salt/modules/test.py]: Sorry about that, didn't mean to put push that ;)

File: salt/modules/test.py [hunk 2]
```diff
@@ -20,7 +20,6 @@
 
 __proxyenabled__ = ['*']
 
-
```
Comment (thatch45) [inline salt/modules/test.py]: Bah, this will lint error now...
sorry....

**Pull request merged**